cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGenerator.java

```diff
@@ -624,7 +638,7 @@ private void generateTransactionalDelegateBody(GeneratorAdapter mg, Method metho
      * Generates the code block for setting context ClassLoader, calling user handler method
      * and resetting context ClassLoader.
      */
-    private void generateInvokeDelegate(GeneratorAdapter mg, int handler, Method method) {
+    private void generateInvokeDelegate(GeneratorAdapter mg, Method method, int handler, int responder) {
```

Comment:
Why reorder the `method` and `handler` while all you want to do is to add a `responder` parameter?